### PR TITLE
[ET-929] Update template path for help section

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ET-929]

Screen Shot 2020-11-05 at 12 14 10 AM

[ET-929]: https://moderntribe.atlassian.net/browse/ET-929

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -667,15 +667,23 @@ public function add_help_section_extra_content( $help ) {
...
 	 * @param array $plugins
 	 *
 	 * @return array
 	 */
 	public function add_template_updates_check( $plugins ) {
 		$plugins[ __( 'Event Tickets', 'event-tickets' ) ] = [
-			self::VERSION,
-			$this->plugin_path . 'src/views/tickets',
-			trailingslashit( get_stylesheet_directory() ) . 'tribe-events/tickets',
+				self::VERSION,
```
Comment (tr1b0t) [inline src/Tribe/Main.php:678]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: c7ec9fb4f14b5f5f4216f4c8aead59d9</sub>

File: src/Tribe/Main.php [hunk 2]
```diff
@@ -667,15 +667,23 @@ public function add_help_section_extra_content( $help ) {
...
+				$this->plugin_path . 'src/views',
```
Comment (tr1b0t) [inline src/Tribe/Main.php:679]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: 2f2dd8ff694dcb4187462d4605bfd8ae</sub>

File: src/Tribe/Main.php [hunk 3]
```diff
@@ -667,15 +667,23 @@ public function add_help_section_extra_content( $help ) {
...
+				trailingslashit( get_stylesheet_directory() ) . 'tribe/tickets',
```
Comment (tr1b0t) [inline src/Tribe/Main.php:680]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: 1491ff3d6a6bc84a5176ba7318818d0d</sub>

File: src/Tribe/Main.php [hunk 4]
```diff
@@ -667,15 +667,23 @@ public function add_help_section_extra_content( $help ) {
...
+		];
+
+		$plugins[ __( 'Event Tickets - Legacy', 'event-tickets' ) ] = [
+				self::VERSION,
```
Comment (tr1b0t) [inline src/Tribe/Main.php:684]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: d2dfd5c29f385ee35a1f8d67098ddc30</sub>

File: src/Tribe/Main.php [hunk 5]
```diff
@@ -667,15 +667,23 @@ public function add_help_section_extra_content( $help ) {
...
+				$this->plugin_path . 'src/views',
```
Comment (tr1b0t) [inline src/Tribe/Main.php:685]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: 4e107a009aa2c61c8d4fe17c9f7f9051</sub>

File: src/Tribe/Main.php [hunk 6]
```diff
@@ -667,15 +667,23 @@ public function add_help_section_extra_content( $help ) {
...
+				trailingslashit( get_stylesheet_directory() ) . 'tribe-events/tickets',
```
Comment (tr1b0t) [inline src/Tribe/Main.php:686]: :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: 7ad58b337d13d2382d2d1d0afca2e6e4</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 6 error(s)

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs changelog

File: readme.txt [hunk 7]
```diff
@@ -122,6 +122,8 @@ Currently, the following add-ons are available for Event Tickets:
 
 * Fix - Calculation fixed for attendee count percentage column while using RSVP only. [ET-876]
 * Fix - Correct specificity of checkboxes and radio buttons styles to prevent conflicts with other Modern Tribe plugins. [ET-922]
+* Fix - Shared capacity stock changes while updating ticket. [ETP-562]
```
Comment (sc0ttkclark) [inline readme.txt:125]:
```suggestion
* Fix - Ensure shared capacity stock does not reset while updating ticket. [ETP-562]
```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**